frontend/src/components/channels/LDKChannelWithoutPeerAlert.tsx

```diff
@@ -0,0 +1,65 @@
+import { AlertTriangleIcon } from "lucide-react";
+import { Link } from "react-router-dom";
+import ExternalLink from "src/components/ExternalLink";
+import { Alert, AlertDescription, AlertTitle } from "src/components/ui/alert";
+import { useChannels } from "src/hooks/useChannels";
+import { useInfo } from "src/hooks/useInfo";
+import { useNodeDetails } from "src/hooks/useNodeDetails";
+import { usePeers } from "src/hooks/usePeers";
+import { Channel } from "src/types";
+
+export function LDKChannelWithoutPeerAlert() {
+  const { data: info } = useInfo();
+  const { data: peers } = usePeers();
+  const { data: channels } = useChannels();
+
+  if (info?.backendType !== "LDK") {
+    // LND does not show disconnected peers
+    return null;
+  }
+
+  const channelWithoutPeer = channels?.find(
+    (channel) => !peers?.some((peer) => peer.nodeId === channel.remotePubkey)
+  );
+  if (!channelWithoutPeer) {
+    return null;
+  }
+
+  return <ChannelWithoutPeerAlertInternal channel={channelWithoutPeer} />;
+}
+
+function ChannelWithoutPeerAlertInternal({ channel }: { channel: Channel }) {
+  const { data: nodeDetails } = useNodeDetails(channel.remotePubkey);
+  return (
+    <Alert>
+      <AlertTriangleIcon className="h-4 w-4" />
+      <AlertTitle>
+        Channel with peer{" "}
+        {nodeDetails?.alias || channel.remotePubkey.substring(0, 8) + "..."} is
+        not peered
```

Comment:
`Channel with peer X is not peered` sounds wrong, maybe we drop the first `peer`?